legend-engine-language-pure-dsl-service/src/test/java/org/finos/legend/engine/language/pure/grammar/test/TestServiceGrammarRoundtrip.java

```diff
@@ -1151,4 +1151,90 @@ public void testServiceTestSuite()
                 "}\n"
         );
     }
+
+
+    @Test
+    public void testServiceTelemetry() {
+        // check for single test parameter
+        test("###Service\n" +
+                "Service meta::pure::myServiceSingle\n" +
+                "{\n" +
+                "  pattern: 'url/myUrl/';\n" +
+                "  owners:\n" +
+                "  [\n" +
+                "    'ownerName',\n" +
+                "    'ownerName2'\n" +
+                "  ];\n" +
+                "  documentation: 'this is just for context';\n" +
+                "  autoActivateUpdates: true;\n" +
+                "  execution: Single\n" +
+                "  {\n" +
+                "    query: param: String[1]|demo::_NPerson.all()->graphFetch(#{demo::_NPerson{Age,Name}}#)->serialize(#{demo::_NPerson{Age,Name}}#);\n" +
+                "    mapping: meta::myMapping;\n" +
+                "    runtime: meta::myRuntime;\n" +
+                "  }\n" +
+                "  telemetry:\n" +
+                "  {\n" +
+                "    metricGroups:\n" +
+                "    [\n" +
+                "      'group1',\n" +
+                "      'group2'\n" +
+                "    ];\n" +
+                "  }\n" +
+                "}\n");
+
+        // Empty MetricGroups
```

Comment:
empty case is written twice. We should not support it. If empty, we should not display it